specification/components/schemas/codeable/DeathNotificationStatus.yaml

```diff
@@ -0,0 +1,41 @@
+type: object
+description: Death Notification Status.
+properties:
+  coding:
+    type: array
+    minItems: 1
+    maxItems: 1
```

Comment:
is this counting from 0? if not should this be 2?

Reply:
PDS API